### PR TITLE
Edit tags for SciPy 2016 conference

### PR DESCRIPTION
**Pull request by willingc**

### CODE REVIEW

File: scipy-2016/videos/tell-me-something-i-dont-know-analyzing-okcupid-profiles-scipy-2016-matar-haller-juan-shishid.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   ],
   "slug": "tell-me-something-i-dont-know-analyzing-okcupid-profiles-scipy-2016-matar-haller-juan-shishid",
   "speakers": [
-    "Matar Haller, Juan Shishid"
```
Comment (codersquid) [inline scipy-2016/videos/tell-me-something-i-dont-know-analyzing-okcupid-profiles-scipy-2016-matar-haller-juan-shishid.json]: The names need to be split in to a list.

**Comment by Daniel-at-github** [comment]
Any one knows what is the common way to merge a PR like this that is OK but needs a little fix?

I will merge it on a new branch and fix it there.

**Comment by willingc** [comment]
@Daniel-at-github Thanks. Just haven't looped back to fix. I've been swamped.

Do you want me to do it quickly?

**Comment by Daniel-at-github** [comment]
Don't worry, it's merged. I'm doing it in other branch.
I hope that github will simplify little fixes in pull requests in the future. To add agility to pull requests.

Anyway, thanks for the pull requests :smile:

**Comment by willingc** [comment]
@Daniel-at-github Thanks. No problem. Completely agree re: github and small fixes. :sunny:

**Comment by Daniel-at-github** [comment]
Done in #53

**Pull request merged**